src/main/java/seedu/address/logic/commands/lesson/ViewLessonCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.commands.lesson;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Optional;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.commons.Code;
+import seedu.address.model.lesson.Lesson;
+import seedu.address.model.lesson.Type;
+
+public class ViewLessonCommand extends Command {
+    public static final String COMMAND_WORD = "info";
+
+    // TODO: edit these messages
+    public static final String MESSAGE_USAGE = "L " + COMMAND_WORD + ": sample";
```

Comment:
Remove Comment. Maybe change MESSAGE_USAGE to have a clearer keyword instead of `sample`